src/main/resources/static/javascript/mainApp/grouping.controller.js

```diff
@@ -70,7 +70,11 @@
         // Default description message when none is provided
         $scope.noDescriptionMessage = "No description given for this Grouping.";
 
+        // Checkbox list
         $scope.membersInCheckboxList = {};
+        $scope.paginatedPageChange = true;
+        $scope.showToolTip = false;
+        $scope.cbPageSelected = false;
```

Comment:
Let's make some changes to the variable names
- I am pretty sure that is a variable already defined as $scope.showTooltip, I would prefer if we just put the conditional checked boxes > 20 right into the ng-show. 
- We can remove the cb prefix, I think it's clear enough to be just called $scope.pageSelected.